targets/TARGET_TOSHIBA/TARGET_TMPM3HQ/device/cmsis.h

```diff
@@ -0,0 +1,13 @@
+/* mbed Microcontroller Library - CMSIS for TMPM3HQ
```

Comment:
Should be the same license header as in other files targets/TARGET_TOSHIBA/TARGET_TMPM3HQ/gpio_api.c .


Also `cmsis_nvic.h`

Reply:
Updated the license header